### PR TITLE
WIP Github actions for TS SDK CI/CD

### PR DESCRIPTION
**Pull request by jkaster**
First draft of CI for TS SDK

This is currently a draft PR

### CODE REVIEW

**Comment by jkaster** [comment]
CI:TEST

File: .github/workflows/tssdk-ci.yml [hunk 1]
```diff
@@ -19,9 +19,200 @@ on:
...
+env:
+  LOOKERSDK_BASE_URL: https://localhost:20000
+  LOOKERSDK_VERIFY_SSL: false
+  TS_JUNIT_OUTPUT_DIR: results/tssdk
+
+defaults:
+  run:
+    shell: bash
+#    working-directory: /
+
```
Comment (joeldodge79) [inline .github/workflows/tssdk-ci.yml:31]: can we try a run w/out this? I think it's superfluous... 
```suggestion
```

**Review by joeldodge79 — APPROVED** [review]
nice work! one comment you can choose to ignore if you want

**Review by bryans99 — APPROVED** [review]
If Joel says its good, then its good for me

**Review by joeldodge79 — COMMENTED** [review]
👍

**Pull request merged**